expose.sh

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env bash
 
+topdir=$(pwd)
+scriptdir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
 # configuration
 
+# Source configuration file if it exists in the project directory
+if [ -f "$topdir/_config.sh" ]; then
+  . "$topdir/_config.sh"
+fi
+
 site_title=${site_title:-"My Awesome Photos"}
 
 theme_dir=${theme_dir:-"theme1"}
 
 # widths to scale images to (heights are calculated from source images)
 # you might want to change this for example, if your images aren't full screen on the browser side
-resolution=(3840 2560 1920 1280 1024 640)
+resolution=${resolution:-(3840 2560 1920 1280 1024 640)}
```

Comment:
This combination of parameter expansion and array initialization does not work as you expected. As I tried it, I got this syntax error:
```
expose.sh: line 414: [: (3840 2560 1920 1280 1024 640): integer expression expected
```